codebuild/release/validate-staging.yml

```diff
@@ -0,0 +1,32 @@
+version: 0.2
+# 'validate-staging' is an AWS::CodeBuild::Project that validates a CodeArtifact staging before a sonatype release
+
+env:
+  variables:
+    REGION: us-east-1
+    DOMAIN: crypto-tools-internal
+    REPOSITORY: java-ddbec-staging
+  parameter-store:
+    ACCOUNT: /CodeBuild/AccountId
+
+phases:
+    runtime-versions:
+      java: $JAVA_ENV_VERSION
+  pre_build:
+    commands:
+      - export SETTINGS_FILE=$(pwd)/codebuild/release/settings.xml
+      - export CODEARTIFACT_TOKEN=$(aws codeartifact get-authorization-token --domain $DOMAIN --domain-owner $ACCOUNT --query authorizationToken --output text --region ${REGION})
+      - export CODEARTIFACT_REPO_URL=https://${DOMAIN}-${ACCOUNT}.d.codeartifact.${REGION}.amazonaws.com/maven/${REPOSITORY}
+  build:
+    commands:
+      - |
+        mvn verify \
+          -Pcodeartifact \
+          -Dcheckstyle.skip \
+          -Desdk.version=$VERSION \
```

Comment:
These shouldn't be needed, as we are verifying the examples pom in this same repo, which should already be updated with the version we want to test under.

Maybe this means that in the release codebuild that we should check the examples depends on the expected VERSION? Alternatively we could keep this in and update the examples pom to allow taking in a version like this (see what we do for busy-engineers-guide [1])

[1] https://github.com/aws-samples/busy-engineers-document-bucket/blob/master/exercises/java/encryption-context-complete/pom.xml#L19